cfgov/cfgov/urls.py

```diff
@@ -12,6 +12,9 @@
 from wagtail.wagtailadmin import urls as wagtailadmin_urls
 from wagtailsharing import urls as wagtailsharing_urls
 
+from wagtail.wagtailcore import views
```

Comment:
Absolutely. 

Also, I had no idea GitHub had `diff` formatting! That's really cool.